specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,783 @@
+          "format": "int32",
+          "type": "integer"
+        },
+        "replicaRetryLimit": {
+          "format": "int32",
+          "description": "Maximum number of replica retries before failing the whole job.",
+          "type": "integer"
+        },
+        "manualTriggerConfig": {
+          "type": "object",
+          "description": "Manual Trigger configurations. Properties replicaCount and parellelism would be set to 1 by default",
+          "properties": {
+            "replicaCount": {
+              "description": "Number of replicas that will be created when the Container App Job is triggered.",
+              "format": "int32",
+              "type": "integer"
+            },
+            "parallelism": {
+              "description": "Number of parallel jobs that can run at a given time.",
+              "format": "int32",
+              "type": "integer"
+            }
+          }
+        },
+        "scheduleTriggerConfig": {
+          "description": "Scheduled Trigger configurations. Properties completions and parellelism would be set to 1 by default",
+          "required": [
+            "cronExpression"
+          ],
+          "type": "object",
+          "properties": {
+            "replicaCount": {
+              "description": "Number of replicas that will be created when the Container App Job is triggered.",
+              "format": "int32",
+              "type": "integer"
+            },
+            "cronExpression": {
+              "description": "Cron formatted repeating schedule (\"* * * * *\")\nfor cronjobs.",
+              "type": "string"
+            },
+            "parallelism": {
+              "description": "Number of parallel jobs that can run at a given\ntime.",
+              "format": "int32",
+              "type": "integer"
+            }
+          }
+        },
+        "registries": {
+          "description": "Collection of private container registry credentials for containers used by the Container apps job",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RegistryCredentials"
+          },
+          "x-ms-identifiers": [
+            "server"
+          ]
+        },
+        "dapr": {
+          "$ref": "#/definitions/Dapr",
+          "description": "Dapr configuration for the Container App."
+        }
+      }
+    },
+    "ContainerAppsJob": {
+      "type": "object",
+      "description": "Container App Job",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "extendedLocation": {
+          "$ref": "./CommonDefinitions.json#/definitions/ExtendedLocation"
+        },
+        "identity": {
+          "description": "managed identities for the Container App to interact with other Azure services without maintaining any secrets or credentials in code.",
```

Comment:
"Managed identities needed by a container app job to interact with other Azure services to not maintain any secrets or credentials in code.",